vite.config.ts

```diff
@@ -6,10 +6,21 @@ import envCompatible from "vite-plugin-env-compatible";
 import vitePluginImp from "vite-plugin-imp";
 import tsconfigPaths from "vite-tsconfig-paths";
 import path from "path";
+import injectVariablesInHTML from "./config/injectVariablesInHTML";
```

Comment:
I think you forgot to include this file in `tsconfig.node.json`

<img width="1166" alt="Screen Shot 2022-09-09 at 11 19 22 AM" src="https://user-images.githubusercontent.com/47064971/189385026-cdca7478-e7fc-40eb-b792-a811a2f19b85.png">